src/StatusQ/Controls/StatusTabButton.qml

```diff
@@ -34,7 +34,7 @@ TabButton {
 
     contentItem: Item {
         implicitWidth: contentItemGrid.implicitWidth
-        implicitHeight: contentItemGrid.implicitHeight + 11
+        implicitHeight: contentItemGrid.implicitHeight + 15
```

Comment:
@igor-sirotin Never trust design tools on numbers - trust the pixels :D

![Screenshot from 2022-08-09 10-44-24](https://user-images.githubusercontent.com/6445843/183593753-e1700100-e2a2-41a7-bd84-09e7bdd41b5f.png)

Reply:
If I set `lineHeight: 22;  lineHeightMode: Text.FixedHeight` we still need one extra pixel.
![Screenshot from 2022-08-09 13-21-36](https://user-images.githubusercontent.com/6445843/183626223-f40d4d5c-daf2-435c-a0ba-0274e1781016.png)

Reply:
As you can see not only lineHeight is needed but also lineHeightMode, and this is not a multiline text so I guess there is no need to force lineHeight because we still cant use just the number from design as is to get same result.